utils/bedrock.py

```diff
@@ -14,6 +14,7 @@ def get_bedrock_client(
     assumed_role: Optional[str] = None,
     endpoint_url: Optional[str] = None,
     region: Optional[str] = None,
+    mode: Optional[str] = 'data-plane',
```

Comment:
Will the meaning of `mode` be intuitive to customers?

What if we create a flag to determine between `bedrock` vs. `bedrock-runtime`?

Or maybe even this function get's refactored to return both clients?

Reply:
Agree with @ihmaws , mode is confusing for customers. Let have a flag instead.

Reply:
Mode is optional and so api callers need not pass it in -- the idea being if we have more than binary choices then mode as a string might help - else we will in any case default to the new API

Reply:
I have converted it into a bool now, by introducing the variable `runtime`